packages/xo-server/src/xo-mixins/jobs/index.js

```diff
@@ -316,11 +316,12 @@ export default class Jobs {
 
       app.emit('job:terminated', status, job, schedule, runJobId)
     } catch (error) {
-      logger.error(`The execution of ${id} has failed.`, {
+      await logger.error(`The execution of ${id} has failed.`, {
```

Comment:
You need to pass `true` as last parameter to make `logger.error` returns a promise.